docs/tutorial/Basic/Setting Up.mdx

```diff
@@ -75,10 +76,45 @@ Change directory into the repository and install the project dependencies
 
 Alternative set up paths can be just cloning or forking the example repository, or starting from scratch using our [fhenix hardhat plugin](../../devdocs/docs/hardhat), [Gitpod environment](../../devdocs/docs/gitpod) or Github Workspaces
 
-## Starting Localfhenix
+## Configuring and Starting Localfhenix
+In order for us to locally deploy and test contracts we will need to run our own node, for this specific use-case we have a docker image named `localfhenix`.
+For ease of use and a simple setup we have hardhat task `localfhenix:start` that downloads and runs the image in a container for you.
```

Comment:
We should probably have a tip or something here that you need docker installed for this to work

Reply:
Will add it to the pre-requisites with a link to their docs